k8s/helm/yona-server/values.yaml

```diff
@@ -166,10 +134,12 @@ sms:
   
 apple:
   enabled: false
+  keystore_secret_filename: dummy.p12
```

Comment:
Why are you making the file name for the secret a variable? The name for the keystore itself remains "dummy".
I would rather move away from dummy in all scenarios and just vary the content between "real" and "dummy" while the name remains serious.